examples/cat_facts/Cargo.toml

```diff
@@ -12,16 +12,16 @@ rust-version = "1.66"
 
 [workspace.dependencies]
 anyhow = "1.0.86"
-crux_core = "0.8"
-crux_http = "0.9"
-crux_kv = "0.4"
-crux_platform = "0.1"
-crux_time = { version = "0.4", features = ["chrono"] }
-# crux_core = { path = "../../crux_core" }
-# crux_http = { path = "../../crux_http" }
-# crux_kv = { path = "../../crux_kv" }
-# crux_platform = { path = "../../crux_platform" }
-# crux_time = { path = "../../crux_time", features = ["chrono"] }
+# crux_core = "0.8"
+# crux_http = "0.9"
+# crux_kv = "0.4"
+# crux_platform = "0.1"
+# crux_time = { version = "0.4", features = ["chrono"] }
+crux_core = { path = "../../crux_core" }
+crux_http = { path = "../../crux_http" }
+crux_kv = { path = "../../crux_kv" }
+crux_platform = { path = "../../crux_platform" }
+crux_time = { path = "../../crux_time", features = ["chrono"] }
```

Comment:
Did you mean to leave these here?

Reply:
yeah for now. Tests will need to run against the local crates

Reply:
will revert just before I merge to master